tests/integration-tests/configs/release-process.yaml

```diff
@@ -0,0 +1,14 @@
+{%- import 'common.jinja2' as common with context -%}
+---
+test-suites:
+  basic:
+    test_essential_features.py::test_essential_features:
+      dimensions:
+        - regions: ["ap-south-1", "cn-north-1", "us-gov-west-1"]
+          instances: {{ common.INSTANCES_DEFAULT_X86 }}
+          oss: ["rhel8"]
+          schedulers: ["slurm"]
+  pcluster_api:
+    test_api_infrastructure.py::test_api_infrastructure_with_default_parameters:
```

Comment:
and here 
`we're verifying that ECR image required for the API is available in the 3 partitions, and it's possible to communicate with it`